packages/EasyUtils/src/CreditCard/CreditCardNumberValidatorInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyUtils\CreditCard;
+
+interface CreditCardNumberValidatorInterface
```

Comment:
@zeromodule, why we can't use https://symfony.com/doc/current/reference/constraints/CardScheme.html?

Reply:
As I understand, we can't just inject Symfony's `ValidatorInterface` to `CreditCardNumberStringSanitizer` because the code should be cross-platform and work identically in Symfony and Laravel.

Reply:
Got it, thanks.

It is a little dirty solution, but I can't imagine a better one. 😂

Reply:
@zeromodule just in case, please verify that your solution works completely identically to Symfony's `CardSchemeValidator`. This is important not to let some specific card numbers be passed to logs.